spec/rdkafka/producer_spec.rb

```diff
@@ -404,9 +404,9 @@ def call(_, handle)
 
     expect(message.payload).to eq "payload headers"
     expect(message.key).to eq "key headers"
-    expect(message.headers[:foo]).to eq "bar"
-    expect(message.headers[:baz]).to eq "foobar"
-    expect(message.headers[:foobar]).to be_nil
+    expect(message.headers['foo']).to eq "bar"
```

Comment:
Nitpick: could we be consistent about using double quotes for strings?

Reply:
@geoff2k Fixed!